src/ui/map.js

```diff
@@ -1367,6 +1372,7 @@ class Map extends Camera {
         return this.style.getSource(id);
     }
 
+    // eslint-disable-next-line jsdoc/require-returns
```

Comment:
I wasn't sure if it would be worthwhile to describe the return value for methods like these where a return value **exists** but it might not be useful to the user.